src/tests/control/test_events.py

```diff
@@ -684,13 +684,12 @@ def test_delete(self):
         doc = self.get_doc('/control/event/ccc/30c3/subevents/%d/delete' % self.subevent1.pk)
         assert doc.select("button")
         doc = self.post_doc('/control/event/ccc/30c3/subevents/%d/delete' % self.subevent1.pk, {})
-        if self.event1.subevents.count() > 1:  # we created 2 subevents. Number of subevents left after one is deleted
-            assert doc.select(".alert-success")
+        assert doc.select(".alert-success")
         if self.event1.subevents.count() == 1:  # last one, can't be deleted
             doc = self.post_doc('/control/event/ccc/30c3/subevents/%d/delete' % self.subevent2.pk, {})
             assert doc.select(".alert-danger")
+            assert not SubEventItem.objects.filter(pk=self.subevent2.pk).exists()
```

Comment:
Why ``assert not …exists()``? Shouldn't it be ``assert … exists()`` as the deletion should fail? :)

Reply:
👍